packages/aws-cdk-lib/aws-ec2/lib/launch-template.ts

```diff
@@ -450,6 +497,12 @@ export interface LaunchTemplateProps {
    * @default - no placement group will be used for this launch template.
    */
   readonly placementGroup?: IPlacementGroupRef;
+
+  /**
+   * Capacity Reservation specification
+   * @default - No capacity reservation specification
+   */
+  readonly capacityReservationSpecification?: LaunchTemplateCapacityReservationSpecification;
```

Comment:
This goes against our design guidelines: 

> Do not introduce artificial nesting for props. It hinders discoverability and makes it cumbersome to use in some languages (like Java) [awslint:props-flat].


https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md#flat